store/db/migration/prod/LATEST__SCHEMA.sql

```diff
@@ -130,4 +130,23 @@ CREATE TABLE memo_relation (
   related_memo_id INTEGER NOT NULL,
   type TEXT NOT NULL,
   UNIQUE(memo_id, related_memo_id, type)
-);
\ No newline at end of file
+);
+
+-- memo_comment
+CREATE TABLE memo_comment (
+  id INTEGER PRIMARY KEY AUTOINCREMENT,
+  created_ts BIGINT NOT NULL DEFAULT (strftime('%s', 'now')),
+  updated_ts BIGINT NOT NULL DEFAULT (strftime('%s', 'now')),
+  content TEXT NOT NULL DEFAULT '',
+  email TEXT NOT NULL DEFAULT '',
+  website TEXT DEFAULT '',
+  name TEXT NOT NULL DEFAULT 'UNKNOWN',
+  visibility TEXT NOT NULL CHECK (
```

Comment:
If I understand correctly, this should be a table for anonymous comments. If so, shouldn't the "Visibility" field be redundant?

Reply:
这个字段是为后续扩展准备的，虽然是匿名评论，但用户未来应该拥有对某条评论可见性编辑的能力，所以该字段预留。
This field is for the subsequent extension. Although it is an anonymous comment, the user should have the ability to edit the visibility of a comment in the future, so this field is reserved.

Reply:
`the user` refers to the memo creator, right? I think this should follow by `memo.visibility`, similar to `memo`<->`resource`. https://github.com/usememos/memos/blob/main/api/v1/resource.go#L610

Also, if we don't use this field now, it would be better to add it later.

Reply:
ok,finished